crates/build/src/ops/ops-log-schema.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I was thinking it might be nice to allow users to reference those schemas using their github URLs. If someone were writing a derivation that was keeping logs in a register, for example, then they could reference the schema in our repo. That might be a little too hacky long term, but it seems preferable to publishing the schemas somewhere else, at least for now.

Reply:
Yea. Let's table this for now, but I'm thinking of this strategy for API models also. Generating the schema and including it as committed content could be part of the build -- it's not either / or.